test/support/conn_case.ex

```diff
@@ -23,6 +23,22 @@ defmodule CompanionWeb.ConnCase do
 
       # The default endpoint for testing
       @endpoint CompanionWeb.Endpoint
+
+      def session_conn() do
```

Comment:
I remember excluding `test/support` from credo and such on the basis that it's boilerplate from Phoenix.

We should probably either drop those exclusions (if this is the right place for this code) or move the code into a module that isn't excluded.